Synthesis failed for java-pubsublite

Hello! Autosynth couldn't regenerate java-pubsublite. :broken_heart:

Here's the output from running `synth.py`:

```
15:41:06,693 autosynth [DEBUG] > Running: git checkout 01b6f23d24b27878b48667ce597876d66b59780e
Note: checking out '01b6f23d24b27878b48667ce597876d66b59780e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 01b6f23 fix(python): install testutils from pypi (#503)
2020-05-19 15:41:06,703 autosynth [DEBUG] > Running: git branch -f autosynth-self-1
2020-05-19 15:41:06,708 autosynth [DEBUG] > Running: git checkout autosynth-self-1
Switched to branch 'autosynth-self-1'
2020-05-19 15:41:06,714 autosynth [INFO] > Running synthtool
2020-05-19 15:41:06,714 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-19 15:41:06,717 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-19 15:41:06,942 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-05-19 15:41:07,196 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-19 15:41:07,240 autosynth [ERROR] > Synthesis failed
2020-05-19 15:41:07,240 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b9371b1 fix: fix the regex in Versions.java to actually split on '.' (#57)
2020-05-19 15:41:07,247 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-05-19 15:41:07,253 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-05-19 15:41:07,543 autosynth [DEBUG] > Running: git checkout b9371b1ef1c7f4ff9b3d86f00d899a1ba3bfd85c
Note: checking out 'b9371b1ef1c7f4ff9b3d86f00d899a1ba3bfd85c'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b9371b1 fix: fix the regex in Versions.java to actually split on '.' (#57)
2020-05-19 15:41:07,550 autosynth [DEBUG] > Running: git checkout d2364eb80b840a36136c8ce12f1c6efabcc9600e
Previous HEAD position was 01b6f23 fix(python): install testutils from pypi (#503)
HEAD is now at d2364eb chore(python): remove one remaining extra bash line (#565)
2020-05-19 15:41:07,565 autosynth [DEBUG] > Running: git branch -f autosynth-40
2020-05-19 15:41:07,569 autosynth [DEBUG] > Running: git checkout autosynth-40
Switched to branch 'autosynth-40'
2020-05-19 15:41:07,576 autosynth [INFO] > Running synthtool
2020-05-19 15:41:07,576 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-19 15:41:07,579 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-19 15:41:07,796 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-40
nothing to commit, working tree clean
2020-05-19 15:41:08,038 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-19 15:41:08,088 autosynth [ERROR] > Synthesis failed
2020-05-19 15:41:08,088 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b9371b1 fix: fix the regex in Versions.java to actually split on '.' (#57)
2020-05-19 15:41:08,095 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-19 15:41:08,101 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/e8afa0af-9d3b-4e3e-9ef4-32b759d46496).

Autosynth is still having trouble generating java-pubsublite. :sob:

Here's the output from running `synth.py`:

```
-b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 7404ee0 chore: release 0.1.4 (#65)
2020-05-20 09:30:04,917 autosynth [DEBUG] > Running: git checkout 01b6f23d24b27878b48667ce597876d66b59780e
Note: checking out '01b6f23d24b27878b48667ce597876d66b59780e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 01b6f23 fix(python): install testutils from pypi (#503)
2020-05-20 09:30:04,929 autosynth [DEBUG] > Running: git branch -f autosynth-self-1
2020-05-20 09:30:04,933 autosynth [DEBUG] > Running: git checkout autosynth-self-1
Switched to branch 'autosynth-self-1'
2020-05-20 09:30:04,952 autosynth [INFO] > Running synthtool
2020-05-20 09:30:04,952 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-20 09:30:04,955 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-20 09:30:05,198 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-05-20 09:30:05,578 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-20 09:30:05,622 autosynth [ERROR] > Synthesis failed
2020-05-20 09:30:05,622 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7404ee0 chore: release 0.1.4 (#65)
2020-05-20 09:30:05,629 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-05-20 09:30:05,635 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-05-20 09:30:05,770 autosynth [DEBUG] > Running: git checkout 7404ee0aa8dbe38bba9310c437668a19ab5b23bb
Note: checking out '7404ee0aa8dbe38bba9310c437668a19ab5b23bb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 7404ee0 chore: release 0.1.4 (#65)
2020-05-20 09:30:05,777 autosynth [DEBUG] > Running: git checkout 520238f0b4d50ead1a20818855f7d732cb442540
Previous HEAD position was 01b6f23 fix(python): install testutils from pypi (#503)
HEAD is now at 520238f chore: add go-apiary config (#571)
2020-05-20 09:30:05,792 autosynth [DEBUG] > Running: git branch -f autosynth-42
2020-05-20 09:30:05,797 autosynth [DEBUG] > Running: git checkout autosynth-42
Switched to branch 'autosynth-42'
2020-05-20 09:30:05,803 autosynth [INFO] > Running synthtool
2020-05-20 09:30:05,803 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-20 09:30:05,805 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-20 09:30:06,034 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-42
nothing to commit, working tree clean
2020-05-20 09:30:06,289 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-20 09:30:06,331 autosynth [ERROR] > Synthesis failed
2020-05-20 09:30:06,331 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7404ee0 chore: release 0.1.4 (#65)
2020-05-20 09:30:06,338 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-20 09:30:06,344 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/e1a3e905-8b24-4bda-895c-1dffe544b2cb).

Autosynth is still having trouble generating java-pubsublite. :sob:

Here's the output from running `synth.py`:

```
ch-name>

HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-20 15:40:35,742 autosynth [DEBUG] > Running: git checkout 01b6f23d24b27878b48667ce597876d66b59780e
Note: checking out '01b6f23d24b27878b48667ce597876d66b59780e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 01b6f23 fix(python): install testutils from pypi (#503)
2020-05-20 15:40:35,751 autosynth [DEBUG] > Running: git branch -f autosynth-self-1
2020-05-20 15:40:35,754 autosynth [DEBUG] > Running: git checkout autosynth-self-1
Switched to branch 'autosynth-self-1'
2020-05-20 15:40:35,770 autosynth [INFO] > Running synthtool
2020-05-20 15:40:35,770 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-20 15:40:35,772 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-20 15:40:35,966 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-05-20 15:40:36,206 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-20 15:40:36,239 autosynth [ERROR] > Synthesis failed
2020-05-20 15:40:36,239 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-20 15:40:36,268 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-05-20 15:40:36,273 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-05-20 15:40:36,413 autosynth [DEBUG] > Running: git checkout e364772a8f9de194fc81d0379d1ce395083643ac
Note: checking out 'e364772a8f9de194fc81d0379d1ce395083643ac'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-20 15:40:36,419 autosynth [DEBUG] > Running: git checkout 7ee92820e64c0aea379781b82399d6b3f3c8655f
Previous HEAD position was 01b6f23 fix(python): install testutils from pypi (#503)
HEAD is now at 7ee9282 fix: don't fail when clean fails (#574)
2020-05-20 15:40:36,432 autosynth [DEBUG] > Running: git branch -f autosynth-43
2020-05-20 15:40:36,435 autosynth [DEBUG] > Running: git checkout autosynth-43
Switched to branch 'autosynth-43'
2020-05-20 15:40:36,439 autosynth [INFO] > Running synthtool
2020-05-20 15:40:36,440 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-20 15:40:36,441 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-20 15:40:36,635 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-43
nothing to commit, working tree clean
2020-05-20 15:40:36,855 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-20 15:40:36,888 autosynth [ERROR] > Synthesis failed
2020-05-20 15:40:36,888 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-20 15:40:36,894 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-20 15:40:36,899 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/59cb7a6b-6cea-48ee-8325-d8f22ca64b00).

Autosynth is still having trouble generating java-pubsublite. :sob:

Here's the output from running `synth.py`:

```
ch-name>

HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-21 02:44:21,015 autosynth [DEBUG] > Running: git checkout 01b6f23d24b27878b48667ce597876d66b59780e
Note: checking out '01b6f23d24b27878b48667ce597876d66b59780e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 01b6f23 fix(python): install testutils from pypi (#503)
2020-05-21 02:44:21,026 autosynth [DEBUG] > Running: git branch -f autosynth-self-1
2020-05-21 02:44:21,031 autosynth [DEBUG] > Running: git checkout autosynth-self-1
Switched to branch 'autosynth-self-1'
2020-05-21 02:44:21,037 autosynth [INFO] > Running synthtool
2020-05-21 02:44:21,037 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-21 02:44:21,040 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-21 02:44:21,276 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-self-1
nothing to commit, working tree clean
2020-05-21 02:44:21,615 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-21 02:44:21,661 autosynth [ERROR] > Synthesis failed
2020-05-21 02:44:21,662 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-21 02:44:21,669 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-05-21 02:44:21,675 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-05-21 02:44:21,809 autosynth [DEBUG] > Running: git checkout e364772a8f9de194fc81d0379d1ce395083643ac
Note: checking out 'e364772a8f9de194fc81d0379d1ce395083643ac'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-21 02:44:21,845 autosynth [DEBUG] > Running: git checkout 7ee92820e64c0aea379781b82399d6b3f3c8655f
Previous HEAD position was 01b6f23 fix(python): install testutils from pypi (#503)
HEAD is now at 7ee9282 fix: don't fail when clean fails (#574)
2020-05-21 02:44:21,864 autosynth [DEBUG] > Running: git branch -f autosynth-43
2020-05-21 02:44:21,869 autosynth [DEBUG] > Running: git checkout autosynth-43
Switched to branch 'autosynth-43'
2020-05-21 02:44:21,876 autosynth [INFO] > Running synthtool
2020-05-21 02:44:21,876 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-21 02:44:21,879 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-21 02:44:22,114 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-pubsublite/synth.py.
On branch autosynth-43
nothing to commit, working tree clean
2020-05-21 02:44:22,379 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-pubsublite/synth.py", line 25, in <module>
    'samples/install-without-bom/*',
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 362, in common_templates
    ["samples/**/src/main/java/**/*.java", "samples/**/pom.xml"]
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 106, in all_snippets
    for snippet, code in all_snippets_from_file(file).items():
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in all_snippets_from_file
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 90, in <dictcomp>
    for snippet, lines in snippet_lines.items()
  File "/tmpfs/src/github/synthtool/synthtool/gcp/snippets.py", line 47, in _trim_leading_whitespace
    max_leading_spaces = min(leading_spaces)
ValueError: min() arg is an empty sequence
2020-05-21 02:44:22,421 autosynth [ERROR] > Synthesis failed
2020-05-21 02:44:22,422 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at e364772 style: format grpc library and protos (#69)
2020-05-21 02:44:22,429 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-21 02:44:22,435 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/ec355070-bb90-48d8-9aef-c7786c0c16d7).

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -1,0 +1,76 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: ci
+jobs:
+  units:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [7, 8, 11]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: ${{matrix.java}}
+    - run: java -version
+    - run: .kokoro/build.sh
+      env:
+        JOB_TYPE: test
+    - name: coverage
+      uses: codecov/codecov-action@v1
+      with:
+        name: actions ${{matrix.java}}
+  windows:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - run: java -version
+    - run: .kokoro/build.bat
+      env:
+        JOB_TYPE: test
+  dependencies:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - run: java -version
+    - run: .kokoro/dependencies.sh
+  linkage-monitor:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - run: java -version
+    - run: .kokoro/linkage-monitor.sh
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - run: java -version
+    - run: .kokoro/build.sh
+      env:
+        JOB_TYPE: lint
+  clirr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - run: java -version
+    - run: .kokoro/build.sh
+      env:
+        JOB_TYPE: clirr

--- a/.gitignore
+++ b/.gitignore
@@ -24,6 +24,7 @@ hs_err_pid*
 
 # intellij project folder
 .idea/
+*.iml
 
 # maven build directory
 target/

--- a/.kokoro/build.bat
+++ b/.kokoro/build.bat
@@ -1,3 +1,3 @@
 :: See documentation in type-shell-output.bat
 
-"C:\Program Files\Git\bin\bash.exe" github/java-pubsublite/.kokoro/build.sh
+"C:\Program Files\Git\bin\bash.exe" %~dp0build.sh

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -99,7 +99,16 @@ mvn -Penable-samples clean verify
     ```
 
 2. [Activate](#profile-activation) the profile.
-3. Define your samples in a normal Maven project in the `samples/` directory
+3. Define your samples in a normal Maven project in the `samples/` directory.
+
+### Code Formatting
+
+Code in this repo is formatted with
+[google-java-format](https://github.com/google/google-java-format).
+To run formatting on your project, you can run:
+```
+mvn com.coveo:fmt-maven-plugin:format
+```
 
 ### Profile Activation
 

--- a/samples/snippets/src/main/java/pubsublite/DeleteTopicExample.java
+++ b/samples/snippets/src/main/java/pubsublite/DeleteTopicExample.java
@@ -72,4 +72,4 @@ public class DeleteTopicExample {
       throw statusException;
     }
   }
-} // [START pubsublite_delete_topic]
+} // [END pubsublite_delete_topic]

--- a/synth.metadata
+++ b/synth.metadata
@@ -4,14 +4,14 @@
       "git": {
         "name": ".",
         "remote": "git@github.com:googleapis/java-pubsublite.git",
-        "sha": "579eaab949bf7e92ae3fc5c877fb3b7bb3370632"
+        "sha": "2faceb03ec7c91c07b1353d74b244aa763140ab0"
       }
     },
     {
       "git": {
         "name": "synthtool",
         "remote": "https://github.com/googleapis/synthtool.git",
-        "sha": "01b6f23d24b27878b48667ce597876d66b59780e"
+        "sha": "7ee92820e64c0aea379781b82399d6b3f3c8655f"
       }
     }
   ]

--- a/synth.py
+++ b/synth.py
@@ -19,6 +19,7 @@ import synthtool.languages.java as java
 # TODO: generate library protos when publicly available
 
 java.common_templates(excludes=[
+  '.kokoro/*/samples.cfg',
   # TODO: allow README generation when public
   'README.md',
   # TODO: allow when pubsublite is available in libraries-bom

